ravendb/tests/jvm_migrated_tests/client_tests/streaming_tests/test_query_streaming.py

```diff
@@ -75,15 +76,17 @@ def test_can_stream_query_results_with_query_statistics(self):
         self.wait_for_indexing(self.store)
 
         with self.store.open_session() as session:
+
+            def __stats_callback(statistics: StreamQueryStatistics):
+                self.assertEqual("Users/ByName", statistics.index_name)
```

Comment:
also, check if the program runs this code

e.g. set flag callback_ran to `True` and check at the end of the test if the `callback_ran` is True, not False